ee/dtr/index.md

```diff
@@ -6,61 +6,41 @@ keywords: registry, repository, images
 
-DTR has the ability to [clean up unreferenced manifests and layers](admin/configure/garbage-collection.md).
+## Built-in Access Control
 
-## Built-in access control
+DTR uses the [same authentication mechanism](https://docs.docker.com/ee/ucp/#built-in-security-and-access-control) as the Universal Control Plane, with which users can be managed manually or synchronized from LDAP or Active Directory. DTR employs [Role Based Access Control](admin/manage-users/index.md) (RBAC), which allow the implementation of fine-grained access control policies for Docker images.
 
-DTR uses the same authentication mechanism as Docker Universal Control Plane.
-Users can be managed manually or synchronized from LDAP or Active Directory. DTR
-uses [Role Based Access Control](admin/manage-users/index.md) (RBAC) to allow you to implement fine-grained
-access control policies for your Docker images.
+## Security Scanning
 
-## Security scanning
+A security scanner is built into DTR, which can be used to discover the versions of the software that is in use in your images. This tool scans each layer and aggregates the results, offering a complete picture of what is being shipped as a part of your stack. Most importantly, as the security scanner is kept up-to-date by tapping into a [periodically updated](admin/configure/set-up-vulnerability-scans.md) vulnerability database, it is able to provide [unprecedented insight into your exposure to known security threats](user/manage-images/scan-images-for-vulnerabilities.md).
 
-DTR has a built-in security scanner that can be used to discover what versions
-of software are used in your images. It scans each layer and aggregates the
-results to give you a complete picture of what you are shipping as a part of
-your stack. Most importantly, it correlates this information with a
-vulnerability database that is kept up to date through [periodic
-updates](admin/configure/set-up-vulnerability-scans.md). This
-gives you [unprecedented insight into your exposure to known security
-threats](user/manage-images/scan-images-for-vulnerabilities.md).
+## Image Signing
 
-## Image signing
-
-DTR ships with [Notary](/notary/getting_started.md)
-built in so that you can use
-[Docker Content Trust](/engine/security/trust/content_trust.md) to sign
-and verify images. For more information about managing Notary data in DTR see
-the [DTR-specific notary documentation](user/manage-images/sign-images/index.md).
+DTR ships with [Notary](/notary/getting_started.md) built in, which allows [Docker Content Trust](/engine/security/trust/content_trust.md) to be be put to use in image signing and verification. For more information about managing Notary data in DTR, refer to the [DTR-specific notary documentation](user/manage-images/sign-images/index.md).
```

Comment:
s/be be/be

Reply:
Do you need "built-in"? 
Notary is included in DTR and it allows you to sign and verify images using [Docker Content Trust](/engine/security/trust/content_trust.md). 
I don't see the [DTR-specific notary documentation] topic. I wonder if this is a redirect from another page. If it is, consider removing the sentence and the link

Reply:
Edits made (with accompanying changes to the topic title for the xref and in the toc.yaml file).